doc/protocol/support.md

```diff
@@ -1,68 +1,87 @@
-| Update                        |             x                 ||
-|   - Authorized Update         |             x                 | Authorization must be performed as standalone call, do not support "in-lining" authorization in update call       |
-| Subscribe                     |             x                 ||
-|   - Authorized Subscribe      |             x                 | Authorization must be performed as standalone call, do not support "in-lining" authorization in subscribe call    |
-|   - Curve Logging Subscribe   |             o                 ||
-|   - Range Subscribe           |             o                 ||
-|   - Change Subscribe          |             o                 ||
-| Unsubscribe                   |             x                 ||
-| Subscription                  |             x                 ||
-| Error messages                |             x                 ||
-| Timestamps                    |             x                 ||
-
-x = implemented; x/o = partly implemented; o = not implemented
-
-For a more detailed view of the supported JSON-schemas [click here](https://github.com/eclipse/kuksa.val/blob/master/kuksa-val-server/include/VSSRequestJsonSchema.hpp)
-
-
-### KUKSA.val databroker gRPC API
-The VISS Standard is not applicable for gRPC protocol. Here is an overview what the gRPC API in KUKSA.val databroker is capable of:
-
-  * Read: Reading VSS datapoints 
-    * Reading current or target value for actuators
-    * Reading some metadata information from VSS datapoints
-  * Write: Writing VSS datapoints
-    * Writing sensor values
-    * Writing current or target value for actuators
-    * Soon: Writing some metadata information from VSS datapoints
-  * Subscription: Subscribing VSS datapoints
-    * Subscribing sensor values
-    * Subscribing current or target value for actuators
-
-
+## Supported protocols
+
+This file contains an overview what the KUKSA.val Server and databroker each supports. It focuses on gRPC and VISS support and also what feeders are supported.
+
+
+| Protocol                  | KUKSA.val server | KUKSA.val databroker |
+|---------------------------|:----------------:|:--------------------:|
+| VISS V1                   |         -        |           -          |
+| VISS V2                   |        x/-       |          x/-         |
+| gRPC (kuksa)              |         x        |           -          |
+| gRPC (kuksa.val.v1)       |         -        |           x          |
+| gRPC (sdv.databroker.v1)  |         -        |           x          |
+
+x = supported; x/- = partially supported; - = not supported
+
+
+### VISSv2 support (websocket transport)
+
+| Feature                       | KUKSA.val server  | KUKSA.val databroker |
+|-------------------------------|:-----------------:|:--------------------:|
+| Read                          |                   |                      |
+|   - Authorized Read           | x<sup>1,2</sup>   |           x          |
```

Comment:
Would it be an idea to duplicate the "Authorized Read" and "Authorized Update" lines so that we have lines for:

- Authorized Read of current value (supported by both)
- Authorized Read of target value (only supported by server)
- Authorized Update of current value (only supported by server)
- Authorized Update of target value (supported by both)

To make it more clear what we have in server WS protocol